### ISSUE TITLE
Is there a way to avoid going to the next gallery?

### ISSUE THREAD

**Issue by Shelob9** [dimsemenov/Touchfolio#22]
In the theme if you use the arrow keys, or the arrows on the sides of the images in the gallery it will skip from the last image in a gallery to the first image in a next gallery. Is there anyway to have it cycle back to the begging of the gallery instead? I have built a menu for moving between galleries.

**Comment by mabok** [comment]
i agree to include this feature as well.

**Comment by rxuriguera** [comment]
+1

**Comment by karimphoto** [comment]
YES! Where can we disable this? It would be great to enable this. Or should I say disable this;)
